cpp/include/raft/matrix/gather.cuh

```diff
@@ -30,10 +30,13 @@ namespace raft::matrix {
 /**
  * @brief  gather copies rows from a source matrix into a destination matrix according to a map.
  *
```

Comment:
Could you please add a small use-case example in the docs for each new function in the public namespace?

Reply:
I assume you were referring to `build_clusters` and `calc_centers_and_sizes` in the other header? I haven't added any new function to `gather.cuh`.